src/main/webapp/app/exam/manage/exams/exam-update.component.html

```diff
@@ -8,8 +8,10 @@ <h4>{{ 'artemisApp.examManagement.createExam' | artemisTranslate }}</h4>
             <ng-template #edit>
                 <h4>{{ 'artemisApp.examManagement.editExam' | artemisTranslate }}</h4>
             </ng-template>
-
+            <!-- Section exam configuration -->
             <div>
+                <hr />
+                <h5 style="padding-bottom: 5px">{{ 'artemisApp.examManagement.sections.configuration' | artemisTranslate }}</h5>
```

Comment:
To include some pedantic feedback: please remove the inline styling here. Consider using [bootstrap classes for spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/) if you do not want to declare an own CSS class.